src/renderer/components/Toolbar.vue

```diff
@@ -169,7 +169,9 @@ export default {
           return true
         }
       } catch (e) {
-        self.openDialog('error', e)
+        if (e.code !== ERR_USER_CANCEL)) {
```

Comment:
Invalid closing parenthesis.

Reply:
Ok, got it. 
But the Toolbar doesn't appear on the UI, what should I do to let it show on UI for testing the function on it ?

Reply:
Thank you for your fix!

I'm sorry. In the current development branch the toolbar is inactive. I'm planning to revitalize by improving future functions.

Reply:
Got it. I am expecting that.